scripts/palma-browser

```diff
@@ -8,12 +8,25 @@ killbrowser() {
     [ -d "$2" ] && rm -rf "$2"
 }
 
+install_chromium_extension() {
+    EXTDIR="${BROWSER_PROFILE}/Default/Extensions"
+    mkdir -p $EXTDIR
+    if [ ! -d "$EXTDIR/$1" ]; then
+        if [ ! -f "/tmp/$1.crx" ]; then
+            CHROMIUM_VERSION=$(chromium --version | awk '{print $2}')
+            wget "https://clients2.google.com/service/update2/crx?response=redirect&acceptformat=crx2,crx3&prodversion=$CHROMIUM_VERSION&x=id%3D$1%26installsource%3Dondemand%26uc" -O /tmp/$1.crx
+        fi
+        unzip /tmp/$1.crx -d $EXTDIR/$1
+    fi
+}
+
```

Comment:
This is mostly cosmetic but we could `rm -f /tmp/$1.crx` after unzipping the extension.

Reply:
It was actually a deliberate choice not to delete the file, so that the next instance of palma-browser does not need to download the same extension again.

Reply:
But in line 14 (`if [ ! -d "$EXTDIR/$1" ]; then`) it checks if the extension is already installed and would therefore not download it again...or am I missing something here ?